src/runtime/recipe/slot-connection.ts

```diff
@@ -113,44 +113,48 @@ export class SlotConnection implements Comparable<SlotConnection> {
   }
 
   isResolved(options?): boolean {
+    const error = (label: string) => {
+      if (options && options.errors) {
+        options.errors.set(this.name, label);
+      }
+      if (options && options.details) {
+        options.details = label; // TODO use .errors instead.
```

Comment:
I had left this off as I do not believe that this is the responsibility of any one engineer. I think we all will benefit from being consistent in the way that we record errors. Will add it so that someone is tracking it (me).